extensions/skyportal/static/js/components/TNSInfo.jsx

```diff
@@ -21,23 +21,23 @@ const TNSInfo = ({ objID }) => {
     return <>Fetching TNS data...</>;
   }
   const objTnsInfo = tnsInfo[objID];
+
   return (
     <span>
-      {objTnsInfo?.name ? (
-        typeof objTnsInfo.name === "string" ? (
+      {
+        objTnsInfo !== null && objTnsInfo?.length > 0 ?
+        objTnsInfo.map((TNSMatch) => (
           <a
-            href={`https://www.wis-tns.org/object/${objTnsInfo.name.split(" ")[1]}`}
+            key={TNSMatch.name}
+            href={`https://www.wis-tns.org/object/${TNSMatch.name.split(" ")[1]}`}
             target="_blank"
             rel="noopener noreferrer"
           >
-            {objTnsInfo.name}
+            {`${TNSMatch.name} `}
```

Comment:
```suggestion
            TNSMatch.name
```

Reply:
I wanted a whitespace after the name in case there were multiple matches, which is very rare, but could happen in principle. Do you have a better suggestion for that?